### ISSUE TITLE
Does dreambooth correctly read text from external captions?

### ISSUE THREAD

**Issue by reqwrew** [TheLastBen/fast-stable-diffusion#1464]
![image](https://user-images.githubusercontent.com/123414411/214139897-caf2a9c9-431e-490a-97f2-2332cd8b600e.png)
This is how it is shown to me in the process of training the model.
![image](https://user-images.githubusercontent.com/123414411/214140241-d7d33a1f-54f1-4e05-9f88-ddeaed73bf13.png)
This is what my caption looks like. Is this just a visual bug or not?

**Comment by TheLastBen** [comment]
yes, I set the maximum of words displayed, but that's just the display, the training uses the whole caption

**Comment by Nalwonk** [comment]
I noticed that after it goes through the image set once, the displayed caption is just the instance word, not sure if it's actually reading the whole caption after that.

**Comment by TheLastBen** [comment]
set the text encoder steps to 0 to use the captions the whole training

**Comment by remybonnav** [comment]
even if i set to 0 text encoder training, the captions still disappear from the the display during training

**Comment by TheLastBen** [comment]
try setting the text encoder steps the same as of the unet steps

**Comment by remybonnav** [comment]
my bad, it was actually working if we set text encoder training step to 0. My captions were not copied in the folder and thus it could not used them.
